### ISSUE TITLE
Explore whether a more descriptive 2FA email subject would be helpful

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#597]

**Comment by amazingphilippe** [comment]
This was found during a Fable test, but is not a WCAG failure, removing the a11y tag, and keeping the UX tag.

**Comment by yaelberger-commits** [comment]
Decided not to do at this time. We don't have evidence this is a user need, only that Fable tester had some friction. Closing this card.
